templates/api.hbs

```diff
@@ -32,7 +40,9 @@ export default function(app: Express, impl: t.{{className name}}Api) {
 						{{#unless schema}}
 						return undefined
 						{{else}}
-						{{#ifmatch mediaType.mimeType '/json'}}
+						{{#ifmatch mediaType.mimeType 'multipart/form-data'}}
+						return {{>frag/fromJson schema prefix='v.'}}('body', req)
```

Comment:
Perhaps it would be easier to duplicate `fromJson.hbs` and make a multipart version?